package.json

```diff
@@ -3,8 +3,11 @@
   "version": "0.1.0-rc26",
   "description": "A set of fully customizable React components made with <3 with styled-components",
   "main": "dist/index.js",
+  "module": "dist/esm/index.js",
   "scripts": {
-    "build": "webpack --config webpack/native.config.babel.js && webpack --config webpack/web.config.babel.js",
+    "build": "npm run build:dev && npm run build:prod",
```

Comment:
Don't you think it's better to use `yarn` even if it's the same behavior ? In order to have a consistent pattern in our scripts, and use either `yarn` or `npm` but not both at the same time

Reply:
Done